docs/sdks/roku.md

```diff
@@ -0,0 +1,506 @@
+---
+title: Roku SDK
+description: Learn how to use Featurevisor Roku SDK
+---
+
+BrightScript SDK for Roku. This package is meant to be used with a [kopytko-framework](https://github.com/getndazn/kopytko-framework), however, if you don't use it, you can simply copy all SDK files and their dependencies to your project (version will be prepared in the future, once anyone is interested in using such).
```

Comment:
we can end the first paragraph of a page with `{% .lead %}` to make it appear a bit larger in published site.

example:

```
BrightScript SDK for Roku is meant to be used with [kopytko-framework](https://github.com/getndazn/kopytko-framework). {% .lead %}

However, if you don't use it, you can simply copy all SDK files and their dependencies to your project (version will be prepared in the future, if anyone is interested).
```